pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2365,12 +2363,12 @@ func (c *Cloud) GetVolumeLabels(volumeName KubernetesVolumeID) (map[string]strin
 		return nil, fmt.Errorf("volume did not have AZ information: %q", aws.StringValue(info.VolumeId))
 	}
 
-	labels[kubeletapis.LabelZoneFailureDomain] = az
+	labels[cloudutil.LabelZoneFailureDomain] = az
```

Comment:
What would it take for us to store these in https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kubelet instead?

Reply:
It would surely be a better place.

Reply:
sorry, I just read your comment from the other day about this exact thing

Reply:
yeah I agree it would be a better place but we should discuss with some other SIGs about best direction forward.

Reply:
My understanding is that these belong to `kubelet` and it's not convenient for maintenance if we move them to staging/src/k8s.io/cloud-provider/`. The problem is that currently there's only `config` in `https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kubelet`.